### ISSUE TITLE
Web: Capacitor WebPlugin "FirebaseAnalytics" config object was deprecated in v3 and will be removed in v4.

### ISSUE THREAD

**Issue by ptmkenny** [capacitor-community/firebase-analytics#84]
**Describe the bug**
As stated in the error message:

`Capacitor WebPlugin "FirebaseAnalytics" config object was deprecated in v3 and will be removed in v4.`

**To Reproduce**
1. Install the plugin in a simple Ionic App.
2. Configure Firebase Analytics for the web according to the README. (Use a FirebaseAnalytics object.):
```

/**
 * Platform: Web
 * Configure and initialize the firebase app.
 * @param options - firebase web app configuration options
 * */
FirebaseAnalytics.initializeFirebase({
  apiKey: "...",
  authDomain: "...",
  databaseURL: "...",
  projectId: "...",
  storageBucket: "...",
  messagingSenderId: "...",
  appId: "...",
  measurementId: "...",
});
```

This results in the following warning in the browser's JS console:

`Capacitor WebPlugin "FirebaseAnalytics" config object was deprecated in v3 and will be removed in v4.`

The deprecation is in Capacitor v3:
```

/**
 * Provided for backwards compatibility. Use the registerPlugin() API
 * instead, and provide the web plugin as the "web" implmenetation.
 * For example
 *
 * export const Example = registerPlugin('Example', {
 *   web: () => import('./web').then(m => new m.Example())
 * })
 *
 * @deprecated Deprecated in v3, will be removed from v4.
 */
```

For example, the `Haptics` plugin does it like this:
```

import { registerPlugin } from '@capacitor/core';
const Haptics = registerPlugin('Haptics', {
    web: () => import('./web').then(m => new m.HapticsWeb()),
});
export * from './definitions';
export { Haptics };
```

capacitor: v 3.0
`@capacitor-community/firebase-analytics: 1.0.0-alpha3`

**Expected behavior**
No error occurs.  (Update to v4.)

**Desktop (please complete the following information):**
 - OS: Windows 10: 10.0.19042 Build 19042
 - Browser: Edge 90.0.818.55

**Comment by brownoxford** [comment]
@ptmkenny can you add a link to the relevant documentation for this?

**Comment by ptmkenny** [comment]
@brownoxford Updated with more info. Could not find the correct documentation on the website, but I found the comment in the capacitor code.

**Comment by brownoxford** [comment]
Ah, thank you! I was confused thinking that this was an issue with passing options to `initializeFirebase()`. I see the problem in `web.ts` where `FirebaseAnalyticsWeb` overrides `constructor()` and passes a legacy config object to `super()`. Patching now.

**Comment by brownoxford** [comment]
Ok, this should be fixed with PR #86, can you please test release`v1.0.0-alpha.4`?

**Comment by ptmkenny** [comment]
Thanks!  Yeah, I made a mistake filing the issue initially-- I thought it was Firebase but it turned out to be Capacitor.

I updated to `v1.0.0-alpha.4`, and the error is gone.  Awesome!  Many thanks for the extremely speedy response!
